tests/scripts/lua/management/lib/swagger.lua

```diff
@@ -31,6 +31,9 @@ describe('Testing v2 management interface', function()
         "name": "Hello World API",
         "resources": {
           "/hello": {
+            "cors": {
+              "origin": "true"
```

Comment:
@alexsong93 - is the value cors.enabled or cors.origin ?  in other swagger files, i thought i saw cors.enabled.